django/test/selenium.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals
+
+import sys
+import unittest
+
+from django.test import LiveServerTestCase, tag
+from django.utils.module_loading import import_string
+from django.utils.six import with_metaclass
+from django.utils.text import capfirst
+
+
+class SeleniumTestCaseBase(type(LiveServerTestCase)):
+    browsers = []
+    browser = None
+
+    def __new__(cls, name, bases, attrs):
+        """
+        Dynamically create new classes and add them to the test module
+        when multiple browsers specs are provided (e.g. --selenium=firefox,chrome).
+        """
+        test_class = super(SeleniumTestCaseBase, cls).__new__(cls, name, bases, attrs)
+        # If the test class is either browser specific or a test base we
```

Comment:
browser-specific (add dash)
base, return it (chop we + add comma)
[aside: not entirely sure what the comment means, maybe the above comment above `browser` vs `browser` will help explain]